concent_api/core/message_handlers.py

```diff
@@ -1354,28 +1356,28 @@ def handle_send_subtask_results_verify(
             reason=message.concents.ServiceRefused.REASON.InvalidRequest,
         )
 
-    if Subtask.objects.filter(  # pylint: disable=no-member
-        subtask_id=compute_task_def['subtask_id'],
-        state__in=[
-            Subtask.SubtaskState.VERIFICATION_FILE_TRANSFER.name,  # pylint: disable=no-member
-            Subtask.SubtaskState.ADDITIONAL_VERIFICATION.name,     # pylint: disable=no-member
-        ]
-    ).exists():
-        return message.concents.ServiceRefused(
-            reason=message.concents.ServiceRefused.REASON.DuplicateRequest,
-        )
+    try:
```

Comment:
same here

Reply:
It is to reduce queries to database. I only one ask database for subtask, and then I check it's state- but I check local variable, not database. This is not perfect solution, it will be changed in https://github.com/golemfactory/concent/pull/778 anyway.

Reply:
ok, here it has been reduced by one ;)